build.gradle

```diff
@@ -139,6 +139,9 @@ allprojects {
         // This is also required, to prevent stderr spam starting with
         //   "OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader cl..."
         jvmArgs "-Xshare:off"
+        // This is required for our dangling thread checks at the end of a connector run.
+        // see IntegrationRunner.stopOrphanedThreads
```

Comment:
I looked at this function and I'm none the wiser as to why this `--add-opens` flag is required. This might be out of scope of this PR but if you do know why please take this opportunity to add some more detailed explanations.

Reply:
it's because we use introspection to change the access of the `ThreadLocal.get(Thread)`